web/javascript/torrent-row.js

```diff
@@ -107,6 +107,14 @@ TorrentRendererHelper.formatDL = function (t) {
     return '↓ ' + Transmission.fmt.speedBps(t.getDownloadSpeed());
 };
 
+TorrentRendererHelper.formatETA = function(t)
+{
+        var eta = t.getETA();
+        if (eta < 0 || eta >= (999 * 60 *60))
+                return "na";
```

Comment:
Or just don't display it at all if it's not available (could be paused or something, so no point in displaying).